cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/ConfigurationGroupsCollection/GroupPanel/index.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelActions from '@material-ui/core/ExpansionPanelActions';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import withStyles, { StyleRules } from '@material-ui/core/styles/withStyles';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import If from 'components/If';
+import { useConfigurationGroupState } from 'components/PluginJSONCreator/Create';
+import GroupActionButtons from 'components/PluginJSONCreator/Create/Content/ConfigurationGroupsCollection/GroupActionButtons';
+import GroupInfoInput from 'components/PluginJSONCreator/Create/Content/ConfigurationGroupsCollection/GroupInfoInput';
+import WidgetCollection from 'components/PluginJSONCreator/Create/Content/WidgetCollection';
+import * as React from 'react';
+
+const styles = (): StyleRules => {
+  return {
+    eachGroup: {
+      display: 'grid',
+      gridTemplateColumns: '5fr 1fr',
+    },
+    groupContent: {
+      display: 'block',
+      padding: '0px 0',
```

Comment:
I think this can be abbreviated to just '0'

Reply:
Resolved.